provisioner/roles/manage_ec2_instances/tasks/resources.yml

```diff
@@ -85,9 +85,15 @@
         from_port: 8080
         cidr_ip: 0.0.0.0/0
       - proto: tcp
-        to_port: 8088
-        from_port: 8088
+        to_port: 8081
```

Comment:
what is port 8081 one for, i wonder if we should add descriptions to each line here for others to help work on these, or move this entirely into vars